lib/read.js

```diff
@@ -95,6 +93,20 @@ class ReadSql extends AdapterRead {
         this.timeField = options.timeField ||
             (options.from || options.to || options.last ? 'time' : undefined);
     }
+    
+    addSorting() {
+        if (this.sortFields) {
+            // adds multiple sort orders
+            this.sortFields.forEach((sortObj) => {
```

Comment:
Oh. I thought having all the order by statements together would be easiest to read since this function determines sort order. No?